### DISCUSSION TITLE
Presidio Image Redactor not working in notebook

### DISCUSSION THREAD

**Discussion by SingTeng** [microsoft/presidio#1005] [Q&A] [answered]
Hi all,

I understand that the Predisio image redactor is still in beta and not production ready. (https://microsoft.github.io/presidio/image-redactor/)

I tried to run it in Jupyter notebook on cloud, and it has error as below:

TesseractNotFoundError: tesseract is not installed or it's not in your PATH. See README file for more information.

I've done the pip install presidio-image-redactor.
Am I missing something?
Let me know if this problem can be solved currently and how to solve it.

Thanks.

**Comment by omri374** [comment]
Hi @SingTeng, In addition to the python package, tesseract should be installed locally. See more info here: https://microsoft.github.io/presidio/image-redactor/#installation

**Reply by davidkhala** [reply]
@omri374 The page is not found anymore

**Reply by SharonHart** [reply, marked as answer]
Since we moved org, the page is accessible through:
https://data-privacy-stack.github.io/presidio/image-redactor/#installation
